tmt/steps/prepare/__init__.py

```diff
@@ -65,10 +65,7 @@ def go(self):
     def run_how(self, how, what):
         """ Run specific HOW """
         self.info('Prepare', f"{how} = '{what}", 'yellow')
-        getattr(self,
-            f"how_{how}",
-            lambda: 'generic',
-            )(how, what)
+        getattr(self, f"how_{how}", self.how_generic)(how, what)
```

Comment:
I'll merge 1bf8619e600e17d808430067d887b0e196dc210e, but maybe I'll squash later. I wonder why it's an issue though... it's just formatting change, right?

Reply:
Oh I see the error now, sorry, you're right.